packages/wattpm/test/metrics.test.js

```diff
@@ -0,0 +1,68 @@
+import { on } from 'node:events'
+import { test } from 'node:test'
+import split2 from 'split2'
+import { prepareRuntime } from '../../basic/test/helper.js'
+import { wattpm } from './helper.js'
+import { ok } from 'node:assert'
+
+test('metrics - should return runtime metrics without format opt', async t => {
+  const { root: rootDir } = await prepareRuntime(t, 'main', false, 'watt.json')
+
+  t.after(() => {
+    startProcess.kill('SIGINT')
+    startProcess.catch(() => {})
+  })
+
+  const startProcess = wattpm('start', rootDir)
+
+  for await (const log of on(startProcess.stdout.pipe(split2()), 'data')) {
+    const parsed = JSON.parse(log.toString())
+
+    if (parsed.msg.startsWith('Platformatic is now listening')) {
+      break
+    }
+  }
+
+  const logsProcess = await wattpm('metrics', 'main')
+
+  ok(logsProcess.stdout.includes('"nodejs_version_info"'))
+  ok(logsProcess.stdout.includes('"http_request_duration_seconds"'))
+  ok(logsProcess.stdout.includes('"http_cache_hit_count"'))
+  ok(logsProcess.stdout.includes('"process_cpu_system_seconds_total"'))
+})
+
+test('metrics - should return runtime metrics with text format', async t => {
+  const { root: rootDir } = await prepareRuntime(t, 'main', false, 'watt.json')
+
+  t.after(() => {
+    startProcess.kill('SIGINT')
+    startProcess.catch(() => {})
+  })
+
+  const startProcess = wattpm('start', rootDir)
+
+  for await (const log of on(startProcess.stdout.pipe(split2()), 'data')) {
+    const parsed = JSON.parse(log.toString())
+
+    if (parsed.msg.startsWith('Platformatic is now listening')) {
+      break
+    }
+  }
+
+  const logsProcess = await wattpm('metrics', 'main', '-f', 'text')
```

Comment:
what is `main` here?

Reply:
It's the name of the service, but it's not used, I removed it on [57addba](https://github.com/platformatic/platformatic/pull/3716/commits/57addba873d0d357dac22255b8cea9fd3fb087ca)